crates/ecs/src/lib.rs

```diff
@@ -152,9 +215,15 @@ impl<'systems> Schedule<'systems> for Unordered<'systems> {
         Self(systems)
     }
 
-    fn next_batch(&mut self) -> Vec<&dyn System> {
-        self.0.iter().map(|system| system.as_ref()).collect()
+    fn currently_executable_systems(&mut self) -> Vec<SystemExecutionGuard<'systems>> {
+        self.0
+            .iter()
+            .map(|system| system.as_ref())
+            .map(|system| SystemExecutionGuard::new(system).0)
```

Comment:
I found it a little confusing that SystemExecutionGuard::new(system) did not follow the convention of return a SystemExecutionGuard, but rather a (SystemExecutionGuard, Receiver)-tuple.

Reply:
Do you think renaming to something other than `new` would help clarify? Maybe `create` or something like that.

Reply:
Yes, that would be clearer.